src/contracts/SafientMain.ts

```diff
@@ -102,8 +102,8 @@ export class SafientMain {
    * @param creatorAddress Address of the creator who created the safe offchain
    * @param safeId Id of the safe
    * @param claimType Type of claim the inheritor has go through
-   * @param signalingPeriod Number of days within which the safe creator is willing to send a signal
-   * @param dDay The timestamp after which the beneficiary can directly claim the safe
+   * @param signalingPeriod The time window in milliseconds within which the creator wants to signal the safe in response to a claim on the safe
```

Comment:
Isn't the `signalingPeriod` denoted in in seconds?

Reply:
Yes

Reply:
Please update these @param docs then.